Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -0,0 +1,249 @@
+﻿using HtmlAgilityPack;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Web;
+
+namespace Selenium.Axe
+{
+    public static class HtmlReport
+    {
+        public static void CreateReport(this IWebDriver webDriver, string destination)
```

Comment:
Would be good to update `docs` with documentation for the new API(s)

Reply:
Done